projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -1942,67 +1147,32 @@
       content: '';
       display: inline-block;
       position: absolute;
-      border: variables.$clr_baselineRem_10px solid transparent;
-      @include mixins.css-var(
-        border-right-color,
-        clr-table-border-color,
-        tables-variables.$clr-table-border-color,
-        variables.$clr-use-custom-properties
-      );
-      top: variables.$clr_baselineRem_0_5 - variables.$clr_baselineRem_2px;
+      border: tokens.$cds-global-space-5 solid transparent;
+      border-right-color: tables-variables.$clr-table-border-color;
+      top: tokens.$cds-global-space-6 - tokens.$cds-global-space-2;
       right: 0;
     }
 
     &:after {
       content: '';
       display: inline-block;
       position: absolute;
-      border: variables.$clr_baselineRem_9px solid transparent;
-      @include mixins.css-var(
-        border-right-color,
-        clr-datagrid-popover-bg-color,
-        datagrid-variables.$clr-datagrid-popover-bg-color,
-        variables.$clr-use-custom-properties
-      );
-      top: variables.$clr_baselineRem_0_5 - variables.$clr_baselineRem_1px;
-      right: -1 * variables.$clr_baselineRem_1px;
+      border: tokens.$cds-global-space-5 solid transparent;
+      border-right-color: datagrid-variables.$clr-datagrid-popover-bg-color;
+      top: tokens.$cds-global-space-6 - tokens.$cds-global-space-1;
```

Same as before, but do we really need odd calc?

```suggestion
      top: mixins.baselinePx(11);
```